src/table/filter.js

```diff
@@ -75,31 +75,33 @@ export default function Filter(props: FilterProps) {
         return nextState;
       }}
       content={
-        <TrapFocus>
-          <Heading {...headingProps}>Filter Column</Heading>
-          <Content {...contentProps}>{props.children}</Content>
-          <Footer {...footerProps}>
-            <Button
-              kind={KIND.minimal}
-              size={SIZE.compact}
-              onClick={() => {
-                onSelectAll();
-              }}
-            >
-              Select All
-            </Button>
-
-            <Button
-              kind={KIND.minimal}
-              size={SIZE.compact}
-              onClick={() => {
-                onReset();
-              }}
-            >
-              Reset
-            </Button>
-          </Footer>
-        </TrapFocus>
+        <FocusTrap>
+          <div>
```

Comment:
What is this `div` added? Any element added has to be not overridable.

Reply:
`FocusTrap` requires a single node be passed and cannot be a Fragment so I wrapped this in a div. The older implementation (`TrapFocus`) would wrapper whatever was inside it in an extra div so I figured it was equivalent here.